diff --git a/node_modules/@mui/joy/Input/InputProps.d.ts b/node_modules/@mui/joy/Input/InputProps.d.ts
index 9cc643a..c17b742 100644
--- a/node_modules/@mui/joy/Input/InputProps.d.ts
+++ b/node_modules/@mui/joy/Input/InputProps.d.ts
@@ -38,7 +38,7 @@ export type InputSlotsAndSlotProps = CreateSlotsAndSlotProps<InputSlots, {
     endDecorator: SlotProps<'span', {}, InputOwnerState>;
 }>;
 export interface InputTypeMap<P = {}, D extends React.ElementType = 'div'> {
-    props: P & Pick<React.InputHTMLAttributes<HTMLInputElement>, 'autoComplete' | 'autoFocus' | 'onClick' | 'onChange' | 'onKeyDown' | 'onKeyUp' | 'onFocus' | 'onBlur' | 'defaultValue' | 'value' | 'type' | 'placeholder' | 'readOnly' | 'required' | 'name' | 'id' | 'disabled'> & {
+    props: P & Partial<Pick<React.InputHTMLAttributes<HTMLInputElement>, 'autoComplete' | 'autoFocus' | 'onClick' | 'onChange' | 'onKeyDown' | 'onKeyUp' | 'onFocus' | 'onBlur' | 'defaultValue' | 'value' | 'type' | 'placeholder' | 'readOnly' | 'required' | 'name' | 'id' | 'disabled'>> & {
         /**
          * Class name applied to the root element.
          */
